robot/Cumulus/tests/browser/gift_entry/edit_template.robot

```diff
@@ -0,0 +1,139 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             robot/Cumulus/resources/PaymentPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Enable Gift Entry
+...             Setup Test Data
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Variables ***
+&{method}         Default Value=Check
+&{check}          Default Value=abc11233
+&{date}           Default Value=Today
+&{custom}         Required=checked
+${amount}         50
+${msg}            Automation test
+
+*** Keywords ***
```

Comment:
Do you always add your _Keywords_ before _Test Cases_ ??
Just wondering if we should follow the same order in all repos. In SAL, we add _Keywords_ at the end. I'll make a note of this to bring it up during our Best Practices meeting

Reply:
yes I usually add it before, good idea we should discuss